packages/blade/src/components/Hoverlay/styles.ts

```diff
@@ -9,6 +9,7 @@ const getHoverlayStyles = (
   },
 ): CSSObject => {
   return {
+    pointerEvents: 'none',
```

Comment:
do button clicks or all mouse events work after this?

Reply:
Yes works.